sys-auth/polkit-pkla-compat/polkit-pkla-compat-0.1-r1.ebuild

```diff
@@ -10,22 +10,20 @@ SRC_URI="https://releases.pagure.org/${PN}/${P}.tar.xz"
 LICENSE="LGPL-2"
 SLOT="0"
 KEYWORDS="amd64 ~arm ~ppc x86"
-IUSE=""
 
 RDEPEND=">=dev-libs/glib-2.30
 	>=sys-auth/polkit-0.110"
-DEPEND="${RDEPEND}
+DEPEND="${RDEPEND}"
+BDEPEND="
 	app-text/docbook-xsl-stylesheets
 	dev-libs/libxslt
 	virtual/pkgconfig"
 
-DOCS="AUTHORS NEWS README"
-
 src_install() {
 	default
-	fowners -R root:polkitd /etc/polkit-1/localauthority
+	fowners -R root:polkitd /etc/polkit-1/localauthority || die
```

Comment:
`fowners` is a gentoo built-in, not an external command

Reply:
Oh that's right, thanks :+1: